Package.swift

```diff
@@ -18,7 +18,7 @@ let macOSPlatform: SupportedPlatform
 if let deploymentTarget = ProcessInfo.processInfo.environment["SWIFTTSC_MACOS_DEPLOYMENT_TARGET"] {
     macOSPlatform = .macOS(deploymentTarget)
 } else {
-    macOSPlatform = .macOS(.v10_13)
+    macOSPlatform = .macOS("12.0")
```

Comment:
use const?

Reply:
Do you mean the existing enum case? I was only replicating what SwiftPM already has and thought that one uses strings for a reason. AFAIR @neonichu made those changes in SwiftPM and maybe could clarify the reasoning for raw string preference?